play/.prettierignore

```diff
@@ -0,0 +1,5 @@
+dist
```

Comment:
Would it be better to just use eslint for formatting?

Reply:
eslint doesn't seem to support running in browsers